recipe/run_test.sh

```diff
@@ -6,8 +6,10 @@ test -f $PREFIX/lib/libcusparseLt.so
 
 # skip symbol/compilation tests on non-x86 archs due to the usage of newer glibc symbols
 if [[ $target_platform == "linux-64" ]]; then
+    export CUDA_STUB="$PREFIX/lib/stubs/libcuda.so"
+    
     ${GCC} test_load_elf.c -std=c99 -Werror -ldl -o test_load_elf
-    ./test_load_elf $PREFIX/lib/libcusparseLt.so
+    LD_PRELOAD="$CUDA_STUB" ./test_load_elf $PREFIX/lib/libcusparseLt.so
```

Comment:
@leofang note the additional `LD_PRELOAD`.

If there is a more modern pattern in other feedstocks, please let me know.,